ddpui/auth.py

```
@@ -68,3 +76,86 @@
+
+
+class CustomJwtAuthMiddleware(HttpBearer):
+    def authenticate(self, request, token):
+        # Validate and decode JWT using SimpleJWT's AccessToken
+        token_payload = None
+        try:
+            access_token = AccessToken(token)
+            token_payload = access_token.payload
+        except Exception as err:
+            logger.exception("Invalid or expired token: %s", err)
+            raise HttpError(401, "Invalid or expired token")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)